### ISSUE TITLE
Yugoslavia - J-22 Orao II loadout overhaul

### ISSUE THREAD

**Issue by nuk92** [PygmalionOfCyprus/cmo-db-requests#998]
### DB Selector

DB3K

### User(s)

1652, 2335 J-22 Orao II Yugoslavia and Serbia

### Weapons Carried

-2x AGM-65B Maverick
-8x BR-128 M74 Munja HE 128mm Rockets
-1x 800 liter Drop Tank
--------------------------
-2x AGM-65B Maverick
-2x 500 liter Drop Tank
-1x 800 liter Drop Tank
--------------------------
-2x Grom
-1x Grom Datalink Pod
-1x 800 liter Drop Tank

--------------------------

-4x Durandal [BLU-107]
-1x 800 liter Drop Tank
--------------------------
-2x Durandal [BLU-107]
-2x 500 liter Drop Tank
-1x 800 liter Drop Tank

--------------------------

-4x BL.755 No.1 CB
-1x 800 liter Drop Tank
--------------------------
-2x BL.755 No.1 CB
-2x 500 liter Drop Tank
-1x 800 liter Drop Tank
--------------------------
-6x BL.755 No.1 CB
-1x 800 liter Drop Tank

--------------------------

-12x FAB-100 M80
-1x 800 liter Drop Tank
--------------------------
-12x FAB-100 M80 [Retarded]
-1x 800 liter Drop Tank

--------------------------

-4x FAB-250 M79
-1x 800 liter Drop Tank
--------------------------
-6x FAB-250 M79
-1x 800 liter Drop Tank
--------------------------
-2x FAB-250 M79
-2x 500 liter Drop Tank
-1x 800 liter Drop Tank
--------------------------
-4x FAB-250 M79 [Retarded]
-1x 800 liter Drop Tank

--------------------------

-4x RAB-250 M91 Pre-Frag Proximity Fuze (Air Burst)
-1x 800 liter Drop Tank
--------------------------
-4x FAB-275 M91 FAE [Retarded] Proximity Fuze
-1x 800 liter Drop Tank
--------------------------
-3x FAB-575 M91 FAE [Retarded] Proximity Fuze

--------------------------

-64x BR-20-57 57mm Rockets [HEAT+FRAG]
-1x 800 liter Drop Tank
--------------------------
-16x BR-128 M74 Munja HE 128mm Rockets
-1x 800 liter Drop Tank
--------------------------
-5x PLAB-340 M73 Incendiary Bomb

### Requires Buddy Illumination

No

### Weather Restrictions

Limited All-Weather

### Time-of-Day Restrictions

Day-Only

### Capable of Quick Turnaround

No

### Comments

This request should replace the current J-22 loadouts as they are wrong.

J-22 Orao II don't have AS-7 Kerry (kh-23) Datalinks, Yugoslavia never had AS-7s. Instead they had a domestic command guided Grom missile which was based on the AS-7 Kerry but developed entirely in Yugoslavia. It had a seperate Datalink Pod.

P.S.
Not sure on my quick turnaround selection.

### Sources

Hardpoints:
https://i.imgur.com/PRFCaRr.jpg?1
https://imgur.com/mULggYY

Weapons (post by Gavrinem):
http://www.paluba.info/smf/index.php?topic=20.msg477382#msg477382
starts at 2:00:
https://www.youtube.com/watch?v=gsnEY1WPgGQ

J-22 Orao with AGM-65B Maverick:
https://imgur.com/VCwp4S4
Durandal, BL-755, Maverick, PLAB-350 Incendiary in one picture:
https://www.mycity-military.com/imgs2/114563_45305350_AFM%20January%202003%20-%20Forged%20in%20War%20Dedicated%20to%20Peace_Page_2a.jpg
https://imgur.com/P2q4fk9

16 BR-57-20 57mm and Munja M74 HE 128mm unguided rockets:
https://imgur.com/ElYu5IH
https://imgur.com/HboChnf
https://imgur.com/CYnr5mM

2x 250kg bomb on one pylon:
https://imgur.com/15tztO0

Grom:
https://imgur.com/krOZhM2

Fab-275 FAE:
https://imgur.com/xXZahmY

Fab-575 FAE:
https://imgur.com/mbePeE4

FAB-250 M79 and FAB-100 M80 page (11-13):
https://balkaninsight.com/wp-content/uploads/2020/04/3.pdf

RAB-250 M91:
https://imgur.com/FixZFZx

Drop Tanks:
https://imgur.com/uAtYcO4

https://www.matrixgames.com/forums/viewtopic.php?f=10233&t=243914&p=4504011&hilit=orao#p4504011

**Comment by PygmalionOfCyprus** [comment]
I think, having handled all those weapon requests, that makes all of them and I'm now clear to proceed on this one. Right?

**Comment by nuk92** [comment]
Yes, I just had to add a flare and gsh-23l request to finish it up.
There is another loadout 2x AM-500 naval bottom mines, but I still have to do more research.
Also there was a handfull of J-22s (attack and recon variant) with ELT-156 RWRs, ELT-459 and ELT-555 ecm pods and domestic VRTIO elint pods in the late 80s but I will make a seperate copy request.

**Comment by nuk92** [comment]
updated
